ppml/services/kms-utils/docker/Dockerfile

```diff
@@ -1,45 +1,101 @@
-FROM ubuntu:20.04
-
-ARG EHSM_KMS_BRANCH_VERSION=main
-ARG SPARK_JAR_REPO_URL
 ARG HTTP_PROXY_HOST
 ARG HTTP_PROXY_PORT
 ARG HTTPS_PROXY_HOST
 ARG HTTPS_PROXY_PORT
+ARG JDK_VERSION=8u192
+ARG SPARK_VERSION=3.1.2
+ARG BIGDL_VERSION=2.1.0-SNAPSHOT
+
+# stage.1 java & spark
+FROM ubuntu:18.04 AS builder
```

Comment:
why change to 18.04?

Reply:
There was package dependency error in 20.04 before, so I temporarily used 18.04, and now the package error in 20.04 has been solved, and already reverted to 20.04.